src/algebra/order/nonneg.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Floris van Doorn
 -/
 import algebra.order.archimedean
+import algebra.order.floor
```

Comment:
```suggestion
```

Reply:
Seems harmless to keep it, since it indicates "we're going to mention `floor` in this file"

Reply:
`algebra.order.archimedean` is synonymous with it, really. Archimedeanness is closely to `floor` and `ceil`.